ci-operator/jobs/open-cluster-management/provider-credential-controller/open-cluster-management-provider-credential-controller-main-postsubmits.yaml

```diff
@@ -101,12 +101,9 @@ postsubmits:
     name: branch-ci-open-cluster-management-provider-credential-controller-main-sonar
     spec:
       containers:
-      - args:
-        - -f
-        - Makefile.prow
-        - sonar/go/prow
-        command:
-        - make
+      - command:
+        - export SONAR_GO_TEST_ARGS="-short ./..."
```

Comment:
Is it `./...` (three dots?)

Reply:
Yep. That's what I saw in other examples and what @ray-harris provided in slack.